action.yaml

```diff
@@ -56,6 +56,9 @@ inputs:
     description: path to docker file relative to docker-context. Passed to [docker build push action file input](https://github.com/docker/build-push-action#inputs)
     required: false
     default: Dockerfile
+  docker-build-contexts:
```

Comment:
Since this is being used, does this not need a default?

Reply:
I believe that no default is essentially null or undefined, so it's like the parameter that uses the input is never actually specified.